doctor_visits/delphi_doctor_visits/sensor.py

```diff
@@ -240,4 +242,14 @@ def fit(y_data,
             np.divide((new_rates[include] * (1 - new_rates[include])), den[include]))
 
         logging.debug(f"{geo_id}: {new_rates[-1]:.3f},[{se[-1]:.3f}]")
-        return {"geo_id": geo_id, "rate": new_rates, "se": se, "incl": include}
+
+        included_indices = [x for x in final_sensor_idxs if include[x]]
+
+        def select_idx(my_list):
```

Comment:
whats the advantage of making this its own function?

Reply:
It made more sense in an earlier version of this code.  Removed